glide.yaml

```diff
@@ -2,9 +2,115 @@ package: github.com/openshift/cluster-config-operator
 import:
 - package: github.com/openshift/library-go
   version: master
+- package: github.com/openshift/api
+  version: dedfb47b1ff14bbd03727fa5720f63d8dc7bea5b
```

Comment:
This is the exact commit where these changes were added. If I go any further (for example, pinning to `release-4.1` explicitly) it will also pull in the apiserver changes in https://github.com/openshift/cluster-config-operator/pull/75